### ISSUE TITLE
LESS compiler error when upgrade to 2.69

### ISSUE THREAD

**Issue by gilbertg** [acdvorak/intellij-lessc-plugin#66]
Hi, I got this parse error when upgrading to v2.69 - where before it worked fine. Any advice?

9:31:48 PM LESS CSS Compiler Error
           mixins.less: Parse Error: Unrecognised input (line 561, column 6) near
           .spanX (@index) when (@index > 0) {
           (~".span@{index}") { .span(@index); }
           .spanX(@index - 1);

**Comment by acdvorak** [comment]
You didn't include a complete, usable code sample, so I took my best guess as to what you meant:

**Input LESS file**:

``` less
.span(@index) {
    width: @index * 50px;
}

.spanX (@index) when (@index > 0) {
    (~".span@{index}") { .span(@index); }
    .spanX(@index - 1);
}

.spanX(5);
```

**Expected CSS output**:

``` css
.span5 { width: 250px; }
.span4 { width: 200px; }
.span3 { width: 150px; }
.span2 { width: 100px; }
.span1 { width: 50px; }
```

I could not get the above LESS code to compile using the official `less.js` compiler (`lessc` command) v1.5.0 or v1.6.0.  Both `less.js` versions gave me the exact same error message you did, so I'm guessing they made a backward-incompatible change that is unrelated to this plugin.

Check the [less.js issue tracker](https://github.com/less/less.js/issues) and see if they can help you troubleshoot your issue.

**Comment by gilbertg** [comment]
yes, was deprecated selector interpolation syntax thanks:
https://github.com/less/less.js/issues/1789

On 9 January 2014 00:30, Andrew C. Dvorak notifications@github.com wrote:

> Closed #66 https://github.com/acdvorak/intellij-lessc-plugin/issues/66.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/acdvorak/intellij-lessc-plugin/issues/66
> .

## 

Gilbert Grant

Cell: +27 (0) 82 875 1008 Mail: gilbert@iron-nest.com

http://www.iron-nest.com

_PLEASE CONSIDER THE ENVIRONMENT BEFORE YOU PRINT_
